template/html/common/item_meta.html

```diff
@@ -7,6 +7,11 @@
         <li>
             <time datetime="{{ isodate .entry.Date }}" title="{{ isodate .entry.Date }}">{{ elapsed .user.Timezone .entry.Date }}</time>
         </li>
+        {{ if .user.ShowReadingTime }}
+        <li>
+            <span>{{ timeToRead .entry.Content }} min read</span>
```

Comment:
I just noticed that `min read` is not translated.

Reply:
Do you think it would it be better with `%d min read` or `estimated reading time : %d minutes`?

Reply:
The shorter version seems better: `%d min read`